src/encoded/static/components/typeutils.js

```diff
@@ -108,8 +108,14 @@ BiosampleTable.columns = {
         title: 'Accession',
         display: biosample => <a href={biosample['@id']}>{biosample.accession}</a>,
     },
-    biosample_type: { title: 'Type' },
-    biosample_term_name: { title: 'Term' },
+    'biosample_ontology.classification': {
+        title: 'Type',
+        getValue: item => item.biosample_ontology && item.biosample_ontology.classification,
```

Comment:
For this and the next one, since biosample_ontology is required, you can just use `item.biosample_ontology.classification,` after the arrow. Same for the next two changes.